core/src/subject/SubjectEntity.ts

```diff
@@ -157,6 +160,14 @@ export class SubjectEntity {
 
     await this.#perspective.createSubject(this, this.#baseExpression);
 
+    await this.#perspective.add(
+      new Link({
+        source: this.#source,
+        predicate: "rdf://has_child",
```

Comment:
What if this is part of the constructor as well @fayeed ? If you don't pass any predicate, this is the default, but I think it is helpful to be able to control this.

Reply:
I also wonder if this save() is the right function name, because this creates a new class instance. 
I notice for example here:

https://typeorm.io/#updating-in-the-database

That you still use save() to update the instance, but we have the update() function. 

I think there are two options:

- Rename save() to create()
- save() could be the same as the update() function, just that it checks if there is a class instance of the baseExpression and if not creates a new entry, if it already exist it just updates the values.

Reply:
Yeah I could do that. Also add a option to change the predicate.

Reply:
Hm, I think `save()` is the right name for this and I'm fine it being both create and update. (it means: making sure that after this call the graph stores the local object).

I have a problem with this `rdf://has_child` predicate. Does that really exist, or are we making something up and calling it RDF?. 

But then, why do have this `source` link here in SubjectEntity at all?

Reply:
I will remove the link creatiion from here and we could do this in flux/ override the save method to this